db/migrate/20230503083828_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true, index: true
```

Comment:
- [ ] Same thing here, this column should also be author_id, If you see the ERD diagram you will notice that the column name for referencing user is named author_id